README.md

```diff
@@ -50,6 +50,12 @@ You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localh
 * To disable fedora internal audit capability, edit `install_scripts/config` and change the FEDORA_AUDIT variable to false. The FEDORA_AUDIT_LOCATION can also be changed from its default "/audit", if necessary.
 * `vagrant up`
 
+####Acceptance Testing
+
+One may validate the state of the environment provisioned by the BASH scripts using [beaker](https://github.com/puppetlabs/beaker).
+The acceptance tests (implemented using [server-spec](http://serverspec.org/)) may be executed locally in the following manner:
+* `rake beaker`
```

Comment:
When you say "executed locally" in which directory do you mean? Ideally, this does not add any host-machine requirements... however, when I `vagrant ssh` into the box, `rake` is not found, and `rake1.9.1 beaker` does not appear to work regardless of which directory I am in.

Some more details here would be helpful.